src/utils/api_helper.ts

```diff
@@ -1892,3 +1936,173 @@ export const getUserLocale = async (): Promise<UserLocale> => {
+}
+
+export const deleteQuickPoll = async (pollId: string) => {
+  return await internalFetch(`/secure/quickpoll/${pollId}`, 'DELETE')
+}
+
+export const getQuickPollParticipants = async (pollId: string) => {
+  return await internalFetch(`/secure/quickpoll/${pollId}/participants`)
+}
+
+export const addQuickPollParticipant = async (
+  participantData: AddParticipantRequest
+) => {
+  return await internalFetch(
+    `/secure/quickpoll/${participantData.poll_id}/participants`,
+    'POST',
+    participantData
+  )
+}
+
+export const getQuickPollBySlug = async (slug: string) => {
+  return await internalFetch(`/quickpoll/${slug}`)
+}
+
+export const cancelQuickPoll = async (
+  pollId: string
+): Promise<CancelQuickPollResponse> => {
+  return await internalFetch(`/secure/quickpoll/${pollId}`, 'PATCH')
+}
+
+export const updatePollParticipantAvailability = async (
+  participantId: string,
+  availableSlots: AvailabilitySlot[],
+  timezone?: string
+) => {
+  return await internalFetch(
+    `/quickpoll/participants/${participantId}/availability`,
+    'PATCH',
+    {
+      available_slots: availableSlots,
+      timezone,
+    }
+  )
+}
+
+export const updateGuestParticipantDetails = async (
+  participantId: string,
+  guestName: string,
+  guestEmail: string
+) => {
+  return await internalFetch(
+    `/quickpoll/participants/${participantId}/details`,
+    'PATCH',
+    {
+      guest_name: guestName,
+      guest_email: guestEmail,
+    }
+  )
+}
+
+export const savePollParticipantCalendar = async (
+  participantId: string,
+  email: string,
+  provider: string,
+  payload?: Record<string, unknown>
+) => {
+  return await internalFetch(
+    `/quickpoll/participants/${participantId}/calendar`,
+    'POST',
+    {
+      email,
+      provider,
+      payload,
+    }
+  )
+}
+
+export const getPollParticipantByIdentifier = async (
+  slug: string,
+  identifier: string
+): Promise<QuickPollParticipant> => {
+  return await internalFetch(
+    `/quickpoll/${slug}/participant/${encodeURIComponent(identifier)}`
+  )
+}
+
+export const getQuickPollGoogleAuthConnectUrl = async (
+  state?: string | null
+) => {
+  return await internalFetch<ConnectResponse>(
+    `/quickpoll/calendar/google/connect${state ? `?state=${state}` : ''}`
+  )
+}
+
+export const getQuickPollOffice365ConnectUrl = async (
+  state?: string | null
+) => {
+  return await internalFetch<ConnectResponse>(
+    `/quickpoll/calendar/office365/connect${state ? `?state=${state}` : ''}`
+  )
+}
+
+export const getOngoingQuickPolls = async (
+  limit = QUICKPOLL_DEFAULT_LIMIT,
+  offset = QUICKPOLL_DEFAULT_OFFSET,
+  searchQuery?: string
+): Promise<QuickPollListResponse> => {
+  const params = new URLSearchParams({
+    limit: limit.toString(),
+    offset: offset.toString(),
+  })
+
+  if (searchQuery && searchQuery.trim()) {
+    params.append('searchQuery', searchQuery.trim())
+  }
+
+  return await internalFetch(`/secure/quickpoll/ongoing-polls?${params}`)
```

Comment:
This should be a single function that has status as a parameter, so we don't need to create two functions that essentially do the same thing see my feedback on the api endpoint